src/adapter/graphql/tests/tests/test_gql_metadata_chain.rs

```diff
@@ -78,9 +89,11 @@ async fn metadata_chain_append_event() {
                     }
                 }
                 "#
+                )
+                .replace("<id>", &create_result.dataset_handle.id.to_string())
+                .replace("<content>", &event_yaml.escape_default().to_string()),
             )
-            .replace("<id>", &create_result.dataset_handle.id.to_string())
-            .replace("<content>", &event_yaml.escape_default().to_string()),
+            .data(AccessToken::new(access_token)),
```

Comment:
I hope all these changes can be simplified after we figure out how to pass catalog with token into GQL handler, coz this became quite messy.

Reply:
Simplified, still, we have to attach catalog to the schema post-factum